src/displayapp/screens/Navigation.cpp

```diff
@@ -128,29 +128,29 @@ namespace {
  * Navigation watchapp
  *
  */
-Navigation::Navigation(Pinetime::Applications::DisplayApp* app, Pinetime::Controllers::NavigationService& nav)
-  : Screen(app), navService(nav) {
+Navigation::Navigation(Pinetime::Applications::DisplayApp* app, Pinetime::Controllers::NavigationService& nav, Controllers::MotorController& motorController)
+  : Screen(app), navService(nav), motorController {motorController} {
 
   imgFlag = lv_label_create(lv_scr_act(), nullptr);
   lv_obj_set_style_local_text_font(imgFlag, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, &lv_font_navi_80);
   lv_obj_set_style_local_text_color(imgFlag, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_CYAN);
   lv_label_set_text(imgFlag, iconForName("flag"));
-  lv_obj_align(imgFlag, nullptr, LV_ALIGN_CENTER, 0, -60);
+  lv_obj_align(imgFlag, nullptr, LV_ALIGN_CENTER, 0, -78);
 
   txtNarrative = lv_label_create(lv_scr_act(), nullptr);
   lv_label_set_long_mode(txtNarrative, LV_LABEL_LONG_BREAK);
   lv_obj_set_width(txtNarrative, LV_HOR_RES);
-  lv_label_set_text(txtNarrative, "Navigation");
+  lv_label_set_text(txtNarrative, "Navigation\n\nwatiting for\ninstructions ...");
   lv_label_set_align(txtNarrative, LV_LABEL_ALIGN_CENTER);
-  lv_obj_align(txtNarrative, nullptr, LV_ALIGN_CENTER, 0, 10);
+  lv_obj_align(txtNarrative, nullptr, LV_ALIGN_CENTER, 0, 32);
 
   txtManDist = lv_label_create(lv_scr_act(), nullptr);
   lv_label_set_long_mode(txtManDist, LV_LABEL_LONG_BREAK);
   lv_obj_set_style_local_text_color(txtManDist, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_GREEN);
   lv_obj_set_width(txtManDist, LV_HOR_RES);
   lv_label_set_text(txtManDist, "--M");
   lv_label_set_align(txtManDist, LV_LABEL_ALIGN_CENTER);
-  lv_obj_align(txtManDist, nullptr, LV_ALIGN_CENTER, 0, 60);
+  lv_obj_align(txtManDist, nullptr, LV_ALIGN_CENTER, 0, 110);
```

Comment:
Similarly, this could be aligned to the bottom edge.
```suggestion
  lv_obj_align(txtManDist, nullptr, LV_ALIGN_IN_BOTTOM_MID, 0, 0);
```

Reply:
Thanks for the review, link to PR and suggestions. I modified the code following your suggestions.